pkg/gconfig/json_loader.go

```diff
@@ -0,0 +1,94 @@
+package gconfig
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/common-fate/granted-approvals/pkg/cfaws"
+	"github.com/pkg/errors"
+	"golang.org/x/sync/errgroup"
+)
+
+type SecretGetter interface {
+	Get(ctx context.Context, path string) (string, error)
```

Comment:
`Get` is quite similar to `Load` (the method used in `JSONLoader`) which I feel could be confusing as they have different use cases - loading all config vs retrieving individual secrets

```suggestion
	GetSecret(ctx context.Context, path string) (string, error)
```